apps/statsig/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "statsig",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@contentful/app-sdk": "^4.29.1",
+    "@contentful/f36-components": "^4.77.3",
+    "@contentful/f36-tokens": "4.2.0",
+    "@contentful/field-editor-reference": "^6.4.4",
+    "@contentful/react-apps-toolkit": "1.2.16",
+    "contentful-management": "10.46.4",
+    "emotion": "10.0.27",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
+    "react-scripts": "5.0.1"
+  },
+  "scripts": {
```

Could you please add this `install-ci` script to your app's package.json file?  This will help the CI build pass. Thanks!

```suggestion
  "scripts": {
    "install-ci": "npm ci",
```